### ISSUE TITLE
IModGet,351 error note on 2021.5

### ISSUE THREAD

**Issue by ubergeekseven** [JonasOuellet/zlayermanager#5]
I ran across this plugin and wanted to try it out. I am currently running zbrush 2021.5 and after allowing both exe files to execute and verifying that the interface shows up. When pressing the button to enable the interface, this error shows up. I am not sure if you can do anything to fix this or not. I know that the instructions stated that this is for 2018. I was hoping it would work with the newer ones anyway. 
I tried looking through the txt files in the folders and could not find any with upwards of 351 lines in which there could be an error. I am not familiar with the method used to code these plugins and thought that subm,itting this issue might at least make you aware that it is not functioning in the newer ones.

I can open the interface outside of zbrush but, it does not hook into the program at all when doing it that way. I use xmd to store my brushes and it can be opened at any time and still load the brushes into zbrush once open. I was hoping that it would be similar as long as it opened.

I have attached the screenshot of the error note. it looks like a good plugin and any help in working with layers seems to be needed. 

![image](https://user-images.githubusercontent.com/6400644/105491471-52d41000-5c84-11eb-9238-363dcd5b65bb.png)

**Comment by JonasOuellet** [comment]
Hey, michael ! im glab to hear that you want to try it, Interface Id changed for 2021.5, let met create a build and release it !

**Comment by JonasOuellet** [comment]
I just released a new version v1.5.0 that support  zbrush 2021.5.  Delete all the zlayerfile in you zstartup folder then extract the zlayermanager_2021.5.zip downloaded from the release.  Let me know if it still doesn't work.

**Comment by ubergeekseven** [comment]
Sorry for the slow response. Considering how fast you replied to the issue, I should have looked sooner. I have notifications off apparently.
Anyway, I am testing this now and when i first started zbrush, it took several minutes to open. i loaded my project and it sat for ten minutes before i killed the program. i removed the plugin as directed and reloaded zbrush. it opened as normal and my project opened as well. 
i had to reboot to get the dll file deleted first. i did launch the exes to make sure that defender would allow running it.

i received zero errors this time but, could not use the program. im sorry i cannot add more information than that.
am i doing something wrong here, or any suggestions that may help you?

**Comment by JonasOuellet** [comment]
Do the ui open when you double click on .exe ?

**Comment by ubergeekseven** [comment]
Yes. It does.

**Comment by JonasOuellet** [comment]
and it doesn't when you launch it from zbrush ?

**Comment by ubergeekseven** [comment]
I cannot get to that point when in zbrush. Let me try one more time just in case. I did notice that the newest download does not have the zLayerManager.txt in the folder.  Is that necessary?

**Comment by ubergeekseven** [comment]
ok. Hold off on thinking about anything more. I have no idea what the difference was between what I did before and now but, IT WORKED!

The only difference in what I did that I can think of is this.
Before starting zbrush the first time around, I opened the exe to make sure that it would get through defender. Then I closed the exe. I opened zbrush and it took several minutes to open. Then it all locked up when trying to open my project.

After that. I close zbrush and tried deleting everything to test if I could open my project after removing zlayermanager. I could not delete it because it kept saying that it was in use in zbrush. I could not find any processes or services that could be using it. Because of that, I rebooted. Then I deleted the rest before running zbrush again and loading my project.

This time around, I did not run the exe first because I figured that defender would allow it since I already told it to. I moved everything to the fodler. Started zbrush and it started right away like normal. Then I loaded my project and ran zlayemanager from within zbrush and it opened. All good, thank you very much!
